test/yubico_test.sh

```diff
@@ -0,0 +1,18 @@
+# Requires dfu-utils to be installed.
+# This runs through the steps described in https://github.com/Yubico/libu2f-host documentation
```

Comment:
Usually I run this test suite for every change: https://github.com/google/u2f-ref-code/tree/master/u2f-tests, it is very complete. But having a simple one is a good thing as well.

Reply:
Ah ok I didn't know about that.   I just wanted a simple 'is the key working' test that didn't involve interacting with a website...